components/brave_wallet_ui/utils/routes-utils.ts

```diff
@@ -213,12 +213,15 @@ export const makeSendRoute = (
   const baseQueryParams = {
     chainId: asset.chainId,
     token: asset.contractAddress || asset.symbol.toUpperCase(),
-    account: account.accountId.uniqueKey
+    account: account.accountId.uniqueKey,
```

Comment:
Can we remove this trailing comma? 

```suggestion
    account: account.accountId.uniqueKey
```